index.js

```diff
@@ -0,0 +1,125 @@
+var express = require('express');
+var config = require('config');
+var bunyan = require('bunyan');
+var cors = require('cors');
+var jwt = require('jsonwebtoken');
+
+// INIT
+var app = express();
+
+var Promise = require("bluebird");
+Promise.config({cancellation: true});
+
+var nano = require('nano-blue')(
+  config.get('database.couchdb.dsn')
+);
+
+app.locals.nano = nano;
+
+// CORS
+app.use(cors({
+  methods: ['GET', 'POST', 'PUT', 'DELETE']
+}));
+
+// LOGGER
+var logger = bunyan.createLogger(config.get('logger'));
+app.use(function(req, res, next) {
+  req.logger = logger;
+  next();
+});
+
+// PARSE
+var bodyParser = require('body-parser')
+app.use(bodyParser.json());
+
+// AUTHENTICATION
+var authentication = function(model) {
+  return function(req, res, next) {
+    var authorizationheader = req.get('Authorization');
+    if (authorizationheader === undefined) {
+      res.send(401, 'Missing authorization header');
+    }
+
+    var tokens = authorizationheader.split('Bearer ');
```

Comment:
Hmm, `split` might not be the best choice here, what if a customer header as `MyBearer xxxx`? Can you improve this with a regular express or whatever more solid?